lib/Ix/Validators.pm

```diff
@@ -245,6 +402,12 @@ sub state ($min = -2**31, $max = 2**31-1) {
   };
 }
 
+=func idstr
+
+The given value must be a GUID string (case-insensitive).
```

Comment:
Maybe ... "The given value must be a valid fooId. For now, this means a GUID string, but could change in the future" or something